products/apache-http-server.md

```diff
@@ -17,6 +17,8 @@ releaseDateColumn: true
 
 identifiers:
 -   repology: apache
+-   cpe: cpe:/a:apache:apache_http_server
+-   cpe: cpe:2.3:a:apache:apache_http_server
```

Comment:
```suggestion
-   cpe: cpe:2.3:a:apache:http_server
```

Nothing found on https://services.nvd.nist.gov/rest/json/cpes/2.0?cpeMatchString=cpe:2.3:a:apache:apache_http_server. Using https://services.nvd.nist.gov/rest/json/cpes/2.0?cpeMatchString=cpe:2.3:a:apache:http_server would be better.